scripts/suppr_asan.txt

```diff
@@ -0,0 +1,2 @@
+#suppress getaddrinfo due to internal error on macos
```

Comment:
these files should be in a subdir /scripts/suppression/ and can we put a short README in there saying this is for clang sanitizers.

Reply:
updated